Unix/wsman/wsmanclient.c

```diff
@@ -375,12 +402,14 @@ static MI_Boolean ProcessFaultResponse(WsmanClient *self, Page **data)
             errorMessage = fault.mi_message;
         }
 
+        omiErrorPtr = GetWsmanCimError(errorType, &omiError);
```

Comment:
the use of omiError is not going to work. you are going to have to pass the error information in a different way through to the PostResult -- maybe a dedicated structure that GetWsmanCimError fills in with things like descriptions and errors and probably causes and the likes that the PostResult then just picks up and sends. It is strange to create this error object on the stack and pass it through to the PostResult and it copies one field out into a new object.
We can discuss this more if you want.